examples/crowdfunding/test/failing-tests.js

```diff
@@ -0,0 +1,348 @@
+import {
+  addressToPk,
+  getProgram,
+  stringToBytes,
+  uint64ToBigEndian
+} from '@algorand-builder/algob';
+import { Runtime, StoreAccount, types } from '@algorand-builder/runtime';
+import { assert } from 'chai';
+
+const minBalance = 10e6; // 10 ALGO's
+const initialDonorBalance = minBalance + 60e6;
+const initialCreatorBalance = minBalance + 0.01e6;
+const goal = 7e6;
+
+describe('Failing Scenario', function () {
+  const master = new StoreAccount(1000e6);
+  let creator = new StoreAccount(initialCreatorBalance);
+  let escrow = new StoreAccount(minBalance);
```

Comment:
Please don't do this, it's confusing. We are initializing it later (using `runtime.getLogicSig`), so it doesn't make sense to create a store account here

Reply:
ok